native/packages/react-native-bpk-component-text/src/BpkText.js

```diff
@@ -46,14 +46,28 @@ const styleForTextStyle = (textStyle) => {
     }
   };
 
+  const determineIosFontFamily = (fontWeight) => {
+    let fontName = '.SFUIDisplay';
+    if (Platform.Version >= 11) {
+      fontName = '.SFProDisplay';
```

Comment:
Does setting `sans-serif` like on Android not do the correct thing for 10 and 11?

Reply:
Just had a look and on iOS the font `System` is what we'd want instead of `sans-serif` in this case